src/csharp/Grpc.Tools.CSharpGenerator/Grpc.Tools.CSharpGenerator.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{B266A314-A7E1-4CF0-935B-3029A0754A90}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Grpc.Tools.CSharpGenerator</RootNamespace>
+    <AssemblyName>Grpc.Tools.CSharpGenerator</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
If you target 4.5, then you make a disservice to VS2010 users (if not all of them have died of old age anyway) who will have to install the 4.5 framework just to run the tool. Can you target 4.0? I think all .NET APIs that are used in your code are 4.0-compatible.